NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/AcceptInvite.java

```diff
@@ -54,15 +53,11 @@ public boolean execute(CommandSender sender, String[] args) {
 		}
 		group.addMember(uuid, type);
 		group.removeInvite(uuid, true);
-		checkRecacheGroup(group);
 		PlayerListener.removeNotification(uuid, group);
 		
-		if(NameLayerPlugin.isMercuryEnabled()){
-			MercuryAPI.sendGlobalMessage("removeInvitation " + group.getGroupId() + " " + uuid, "namelayer");
-		}
-		
+		Mercury.message("removeInvitation " + group.getGroupId() + " " + uuid);
+				
 		p.sendMessage(ChatColor.GREEN + "You have successfully been added to the group as a " + type.name() +".");
-		checkRecacheGroup(group);
```

Comment:
Good catch on removing the double recache but same issue as above; checkRecacheGroup seemed to fix most of the synchronization issues we had previously. I'm dubious about removing it so completely.

Also, wrap / externalize Mercury.message as prior.